packages/client/src/ee/containers/DeploymentListContainer/DeploymentCard.tsx

```diff
@@ -61,6 +61,9 @@ interface Props
 }
 
 export function DeploymentCard({ endpoint, ...props }: Props) {
+  if (!props.name) {
```

Comment:
The condition statement should not add before the hook.

Reply:
hmm... got it. I will use another way to fix this issue